### PR TITLE
test(e2e): Refactor home ui test to use page objects

### PR DESCRIPTION
**Pull request by joerter**
@webteckie

### CODE REVIEW

File: test/e2e/adminUI/pages/home.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
+module.exports = {
+	elements: {
+		dashboardHeader: '.dashboard-heading'
+	},
+	sections: {
+		accessGroup: {
+			selector: '.dashboard-group:nth-child(1)',
```
Comment (webteckie) [inline test/e2e/adminUI/pages/home.js]: comment for later:  we should replace positional selectors like this with xpath fixed selectors.

File: test/e2e/adminUI/pages/home.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+					elements: {
+						label: '.dashboard-group__list[data-list-path="users"] .dashboard-group__list-label',
+						plusIconLink: '.dashboard-group__list[data-list-path="users"] a.dashboard-group__list-create.octicon.octicon-plus',
+						itemCount: '.dashboard-group__list[data-list-path="users"] .dashboard-group__list-count',
+					},
+				},
+			},
+		},
+		fieldsGroup: {
+			selector: '.dashboard-group:nth-child(2)',
```
Comment (webteckie) [inline test/e2e/adminUI/pages/home.js]: ditto: comment for later:  we should replace positional selectors like this with xpath fixed selectors.

File: test/e2e/adminUI/pages/home.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+					elements: {
+						label: '.dashboard-group__list[data-list-path="selects"] .dashboard-group__list-label',
+						plusIconLink: '.dashboard-group__list[data-list-path="selects"] a.dashboard-group__list-create.octicon.octicon-plus',
+						itemCount: '.dashboard-group__list[data-list-path="selects"] .dashboard-group__list-count',
+					},
+				},
+			},
+		},
+		otherGroup: {
+			selector: '.dashboard-group:nth-child(3)',
```
Comment (webteckie) [inline test/e2e/adminUI/pages/home.js]: ditto: comment for later:  we should replace positional selectors like this with xpath fixed selectors.

File: test/e2e/adminUI/tests/group002Home/uiTestBooleansTab.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
+var adminUI = require('../../adminUI');
```
Comment (webteckie) [inline test/e2e/adminUI/tests/group002Home/uiTestBooleansTab.js]: why do we still need the adminUI.js here?

File: test/e2e/adminUI/tests/group002Home/uiTestCodeTab.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
+var adminUI = require('../../adminUI');
```
Comment (webteckie) [inline test/e2e/adminUI/tests/group002Home/uiTestCodeTab.js]: ditto: why do we still need the adminUI.js here?

File: test/e2e/adminUI/tests/group002Home/uiTestEmailsTab.js [hunk 6]
```diff
@@ -0,0 +1,35 @@
+var adminUI = require('../../adminUI');
```
Comment (webteckie) [inline test/e2e/adminUI/tests/group002Home/uiTestEmailsTab.js]: ditto:  why do we still need the adminUI.js here?

File: test/e2e/adminUI/tests/group002Home/uiTestNamesTab.js [hunk 7]
```diff
@@ -0,0 +1,35 @@
+var adminUI = require('../../adminUI');
```
Comment (webteckie) [inline test/e2e/adminUI/tests/group002Home/uiTestNamesTab.js]: ditto:  why do we still need the adminUI.js here?

File: test/e2e/adminUI/tests/group002Home/uiTestNumbersTab.js [hunk 8]
```diff
@@ -0,0 +1,35 @@
+var adminUI = require('../../adminUI');
```
Comment (webteckie) [inline test/e2e/adminUI/tests/group002Home/uiTestNumbersTab.js]: ditto: why do we still need the adminUI.js here?

File: test/e2e/adminUI/tests/group002Home/uiTestOtherListsTab.js [hunk 9]
```diff
@@ -0,0 +1,35 @@
+var adminUI = require('../../adminUI');
```
Comment (webteckie) [inline test/e2e/adminUI/tests/group002Home/uiTestOtherListsTab.js]: why do we still need the adminUI.js here?

File: test/e2e/adminUI/tests/group002Home/uiTestSelectsTab.js [hunk 10]
```diff
@@ -0,0 +1,35 @@
+var adminUI = require('../../adminUI');
```
Comment (webteckie) [inline test/e2e/adminUI/tests/group002Home/uiTestSelectsTab.js]: why do we still need the adminUI.js here?

File: test/e2e/adminUI/tests/group002Home/uiTestUsersTab.js [hunk 11]
```diff
@@ -0,0 +1,35 @@
+var adminUI = require('../../adminUI');
```
Comment (webteckie) [inline test/e2e/adminUI/tests/group002Home/uiTestUsersTab.js]: why do we still need the adminUI.js here?

**Comment by webteckie** [comment]
@joerter made some nickpick comments.  I like the separation of groups/tabs.The main comment I have is that like fields/lists we should probably explore this group/tabs pattern a bit more.  Instead of each tab being a test could it be a page object?  You can then attach those tab POs to the home page object and then use the home test to simply command the tabs to do their own test.  I'm happy to merge this as-is if you want to do the extra work as a new PR.  Let me know your preference.

**Comment by joerter** [comment]
@webteckie I think I'd like this to be merged and then I will refactor again into a similar pattern that you have for lists and fields

**Comment by joerter** [comment]
@webteckie Okay I think we're good to go here!

**Pull request merged**